docs/getting_started.md

```diff
@@ -47,6 +47,18 @@ sudo apt install clang gcc gcc-multilib git gettext python3-dev python3-venv
 
 1. Install [brew](https://brew.sh/).
 
+2. Add `brew` Python to your `PATH`:
+
+```shell
+echo "export PATH=\"$(brew --prefix)/opt/python@3/libexec/bin:$PATH\"" >> ~/.zshrc
```

Comment:
Is `brew link` not enough here? I don't recall doing this.

Reply:
hmm maybe, just found this worked, will test this before merging, thx!